docs/user/config-file/examples/sphinx/.readthedocs.yaml

```diff
@@ -10,13 +10,17 @@ build:
   tools:
     python: "3.11"
     # You can also specify other tool versions:
-    # nodejs: "19"
-    # rust: "1.64"
-    # golang: "1.19"
+    # nodejs: "20"
+    # rust: "1.70"
+    # golang: "1.20"
 
 # Build documentation in the "docs/" directory with Sphinx
 sphinx:
-   configuration: docs/conf.py
+  configuration: docs/conf.py
+  # You use the dirhtml builder for cleaner URLs
+  # builder: "dirhtml"
+  # Fail on all warnings to avoid broken references
+  # fail_on_warning: true
```

Comment:
Reasons to add this:

1) I just had a support inquiry where it seems they have somehow been confused about how to apply this option that was previously available through the UI. I think adding it in the example might help.

2) By also mentioning `fail_on_warning`, we have mentioned all options available :) It  is a really good feature to be aware of, which I think most Sphinx projects will benefit from.

Reply:
I think these are good hints and makes sense to mention these config keys here 👍🏼 . I'm not sold on the copy for the comments but I'm not how to suggest something, tho. I'm pinging @ericholscher here who is my English mentor 😛

Reply:
This copy looks good to me 👍